SBAT.md

```diff
@@ -110,17 +112,17 @@ a rare event, and that the generation number for the upstream code base will
 suffice in most cases.
 
 A product-specific generation number is needed if a CVE is fixed in
-code that **only** exists in a specific products branch. This would either
-be something like product-specific patches, or a miss-merge that only
+code that **only** exists in a specific product's branch. This would either
+be something like product-specific patches, or a mis-merge that only
 occurred in that product. Setting a product-specific generation number
 for such an event eliminates the need for other vendors to have to
 re-release the binaries for their products with an incremented global
 number.
 
-However once the global number is bumped for the next upstream CVE fix
-there is no need to carry it any longer since satisfying the check of
-the global number will also exclude any of the older product-specific
-binaries.
+However, once the global number is bumped for the next upstream CVE
+fix there will be no further need to carry that product-specific
+generation number; satisfying the check of the global number will also
+exclude any of the older product-specific binaries.
```

Comment:
The independent clause here makes this feel like a run-on.  Why not just make it two sentences?

Reply:
ACK